src/use_file.rs

```diff
@@ -29,9 +30,11 @@ const FILE_PATH: &str = "/dev/random\0";
 #[cfg(any(target_os = "android", target_os = "linux", target_os = "redox"))]
 const FILE_PATH: &str = "/dev/urandom\0";
 
-pub fn getrandom_inner(dest: &mut [u8]) -> Result<(), Error> {
+pub fn getrandom_inner(dest: &mut [MaybeUninit<u8>]) -> Result<(), Error> {
     let fd = get_rng_fd()?;
-    let read = |buf: &mut [u8]| unsafe { libc::read(fd, buf.as_mut_ptr() as *mut _, buf.len()) };
+    let read = |buf: &mut [MaybeUninit<u8>]| unsafe {
+        libc::read(fd, buf.as_mut_ptr() as *mut _, buf.len())
```

Comment:
BTW, is there a reason why we don't use explicit cast to `*mut core::ffi::c_void` here?

Reply:
I think this was just because we didn't have it before, but I can make the cast explicit.

Reply:
On second thought, maybe it because different targets accept different types in `libc::read`? Seems extremely unlikely, but still worth to check.

Reply:
I try to minimize the changes I make in PRs to what's relevant and necessary. The old code had `buf.as_mut_ptr() as *mut _` so I left the `as *mut _` as it was.